applications/StructuralMechanicsApplication/custom_processes/total_structural_mass_process.cpp

```diff
@@ -61,36 +65,45 @@ void TotalStructuralMassProcess::Execute()
         // We get the values from the condition
         const Properties& this_properties = it_elem->GetProperties();
 
+        element_mass = 0.0; // resetting the element-mass
+
         if (local_space_dimension == 0) { // POINT MASSES
             if (it_elem->Has(NODAL_MASS))
-                total_mass += it_elem->GetValue(NODAL_MASS);
+                element_mass = it_elem->GetValue(NODAL_MASS);
```

Comment:
I don't get the changes, why we move from += to just = and vice versa?

Reply:
bcs now I am computing the mass element-by-element and not summing up the total mass
after computing it element by element I distribute the mass among the nodes and finally sum up the nodal-masses to compute the total mass